suncityla/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "next build && prisma generate",
```

Comment:
I would suggest to make a new script, something like
`build:vercel : "npm run prisma:generate && npm run build`
And then in vercel call `build:vercel`

Reply:
Okay I have updated the file, can you check it out now